components/schema-migrator/migrations/201912131700_add_providername_app.up.sql

```diff
@@ -1,2 +1,8 @@
 ALTER TABLE applications
-    ADD COLUMN provider_display_name varchar(256) NOT NULL;
```

Comment:
create separate migration for renaming.
If someone has this migration (201912131700_add_providername_app.up.sql), it has column provider_display_name and provider_name is missing, but migration won't be executed.

Reply:
I've got question to 2 cases:
 1. somebody has en empty database, so the migration is successful.
 2. somebody has something in database, in this case the old migration will fail.

How we should apply fixing migration?

Changing current migration causes:
1. If somebody has empty database, he has invalid database schema.
2. If somebody has sth in database, he will reapply this migration and the database schema will be in good state.

Adding new migration causes:
1. if somebody has empty database, after migrations the database will have valid schema
2. if somebody has data in db, he won't execute this migration, because the old faulty will fail.